ra_aid/proc/interactive.py

```
@@ -1,90 +1,130 @@
+      RuntimeError: If an error occurs during execution.
     """
-    # Fail early if cmd is empty
+    # Fail early if cmd is empty.
     if not cmd:
         raise ValueError("No command provided.")
-
-    # Check that the command exists
+    
+    # Check that the command exists.
     if shutil.which(cmd[0]) is None:
         raise FileNotFoundError(f"Command '{cmd[0]}' not found in PATH.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)